simularium/simularium-website
# Fix/nav tooltips

Problem
=======
On staging right now, it's hard to click the "share" button because the target is covered by the tooltip target 

Solution
========
Adjusted the tooltip positioning so they don't cover the button target 

## Type of change
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)


Steps to Verify:
----------------
1. load a local simularium file 
2. click "share"
3. should be easer than it is on staging
